workflows/diagnostics/tests/prognostic/emulation/_regtest_outputs/test_single_run.test_compute_summaries[global_average_cloud_5d_300mb_ppm].out

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Is this empty because the test dataset doesn't allow computing this metric? If so maybe add what is needed to the test dataset?

Reply:
Yah. That's correct. The "test" data doesn't have any data for this time point. The coverage of this case is actually pretty useful (it initially failed in the tests) since not all runs survive to 5 days.

Reply:
Makes sense

Reply:
I thought about this some more and writing a mock for this is pretty tricky...suggesting I should refactor the API. I have an idea about how to do this, but would rather not ATM since this is just diagnostics code.

This code does works and will be frequently run as part of our typical workflows so we'll find any bugs pretty quickly.